Doc/faq/general.rst

```diff
@@ -297,7 +297,7 @@ How stable is Python?
 
 Very stable.  New, stable releases have been coming out roughly every 6 to 18
 months since 1991, and this seems likely to continue.  As of version 3.9,
-Python will have a major new release every 12 months (:pep:`602`).
+Python will have a minor new release every 12 months (:pep:`602`).
 
 The developers issue "bugfix" releases of older versions, so the stability of
```

Comment:
@AlexWaygood: I find the use of double quotes strange here. Bugfix releases contain bugfixes, not ... eh, "bugfixes" 😆

Reply:
Heh, agreed, we could probably lose these quotes 😄